src/root/common.tt

```diff
@@ -509,7 +524,19 @@ BLOCK renderJobsetOverview %]
       <th>Name</th>
       <th>Description</th>
       <th>Last evaluated</th>
-      <th colspan="2">Success</th>
+      <th style="width: 2em">
+            Jobs
+      </th>
```

Comment:
I do not have a hydra instance setup with to test the layout, but from reading the code it loos like this would add an additional column with the name "Jobs", and three other columns, while the following changes only splits the existing column in three. This leaves four headers for three columns, possibly putting the successes under jobs, the failures under successes and queued under failed? Tell me if I'm wrong here.

Reply:
Your analysis is not quite correct.

For this page, there were originally two columns: one contained the success percentage value and the other column contained the 2-3 indicators of success count, failure count, and (optional) queued count.  The "Success" title spanned both columns.

The change converts it to 4 columns: the percentage column now has the header "Jobs" and the 3 other columns have the associated markers as headers, with the corresponding counts separated into those three columns.

I'll post a screenshot shortly.

Reply:
Here's the screenshot for this page:
<img width="1381" alt="screen shot 2018-12-15 at 9 06 05 pm" src="https://user-images.githubusercontent.com/787421/50050371-59f3c180-00ad-11e9-9c15-94f22d591dd7.png">

I don't have an account on the nixos hydra to show the corresponding older page, and would need to back out my changes locally, but I'm guessing you can see the existing configuration easily enough from your account.

Reply:
Eek, sorry! Yes, this looks fine!